### ISSUE TITLE
NullPointerException when analysing liquibase-core-4.9.0.jar with Maven plugin 8.1.1

### ISSUE THREAD

**Issue by frangipanes** [dependency-check/DependencyCheck#5523]
**Describe the bug**
Running the dependency-check:aggregate tasks with the maven plugin fails with a NullPointerException when analysing liquibase-core-4.9.0.jar.

**Version of dependency-check used**
The problem occurs using version 8.1.1 of the maven plugin. It does not occur with version 8.1.0.

I think it is related to this change: https://github.com/jeremylong/DependencyCheck/commit/e183bfb47e72cfb315cd128af52dc16b989c4967

**Log file**

```
[WARNING] An unexpected error occurred during analysis of '/Users/user/.m2/repository/org/liquibase/liquibase-core/4.9.0/liquibase-core-4.9.0.jar' (Jar Analyzer): null
[ERROR] 
java.lang.NullPointerException
    at org.owasp.dependencycheck.xml.pom.Model.setGAVFromPomDotProperties (Model.java:399)
    at org.owasp.dependencycheck.analyzer.JarAnalyzer.analyzePOM (JarAnalyzer.java:450)
    at org.owasp.dependencycheck.analyzer.JarAnalyzer.analyzeDependency (JarAnalyzer.java:330)
    at org.owasp.dependencycheck.analyzer.AbstractAnalyzer.analyze (AbstractAnalyzer.java:131)
    at org.owasp.dependencycheck.AnalysisTask.call (AnalysisTask.java:88)
    at org.owasp.dependencycheck.AnalysisTask.call (AnalysisTask.java:37)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:829)
```

**To Reproduce**
1. Create a maven project with a dependency on liquibase-core:4.9.0
2. Include the dependency-check maven plugin at version 8.1.1
3. Run `mvn dependency-check:aggregate`

**Expected behavior**
The dependency-check should complete and render the list of CVEs as usual (if there are any).

**Additional context**
I assume this bug is not specific to liquibase-core-4.9.0.jar, but it's the first dependency that triggered the issue for us.

**Comment by Grimoren** [comment]
A ditto on this for gradle plugin 8.1.1 as well.

**Comment by Grimoren** [comment]
Looks like the fix will be out with 8.1.2:
https://github.com/jeremylong/DependencyCheck/issues/5521
